src/strings/PowerFxResources.en-US.resx

```diff
@@ -4579,4 +4607,32 @@
     <value>Delegation warning. The result of this argument '{0}' may be truncated for large data sets before being passed to the '{1}' function.</value>
     <comment>Error message when an argument to non-delegable function has possible delegation and resulting rows may be truncated</comment>
   </data>
+  <data name="ErrNeedPrimitive" xml:space="preserve">
+    <value>Expected a text, numeric, boolean, or date/time value.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrSummarizeNoGroupBy" xml:space="preserve">
+    <value>Summarize requires at least one grouping column.</value>
+    <comment>Error Message.</comment>
```

Comment:
>Message [](http://example.com/codeflow?start=19&length=7)

Please add `{Locked=Summarize}` to the comment to prevent the translators from localizing the function name. Also mention that this is a function name. Similar comment for other error messages as well.